src/Controller/EnMarche/ApplicationRequestCandidate/AbstractController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche\ApplicationRequestCandidate;
+
+use AppBundle\ApplicationRequest\ApplicationRequestRepository;
+use AppBundle\ApplicationRequest\ApplicationRequestTypeEnum;
+use AppBundle\Controller\CanaryControllerTrait;
+use AppBundle\Entity\ApplicationRequest\ApplicationRequest;
+use AppBundle\Form\ApplicationRequest\ApplicationRequestTagsType;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+abstract class AbstractController extends Controller
```

Comment:
This name can be confuse with the `AbstractController` of the `FrameworkBundle`.

WDYT to rename it ?

Reply:
why not, done